pages/components/Input/input.html

```diff
@@ -0,0 +1,288 @@
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Buttons/button.html">Button</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Cards/cards.html">Card</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Image/image.html">Image</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Input/input.html">Input</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="#">Modal</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="#">Typography</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="#">Toast</a>
+                    </li>
+                </ul>
+            </aside>
+            <main class="docs-main-container">
+                <div class="docs-hero-header">
+                    <h1 class="docs-hero-heading">Input</h1>
+                    <p class="docs-hero-para">
+                        Inputs are way of user's interaction with the website, they are used to get information from the user such as login/signup process and in form elements.<br/><br/>To use any input element add the class <b>input</b>, followed by the variant class names.
+                    </p>
+                </div>
+                <div class="component-variants">
+                    <h3 class="component-varaint-heading">Standard Inputs</h3>
+                    <p class="variant-specifier">
+                        Variants available: <span class="variant-item">std-input</span>
+                    </p>
+                </div>
+                <div class="inputs-preview-container">
+                    <div class="std-input">
+                        <label for="#user-name">Username:</label>
+                        <input id="user-name" placeholder="Enter your username" type="text"/>
+                    </div>
+                    <div class="std-input">
+                        <label for="#password">Password:</label>
+                        <input id="password" placeholder="Enter your password" type="password"/>
+                    </div>
+                    <div class="std-input">
+                        <label for="#mobile-number">Mobile Number:</label>
+                        <input id="mobile-number" placeholder="Enter your mobile number" type="number"/>
+                    </div>
+                    <div class="std-input">
+                        <label for="#user-email">Email id:</label>
+                        <input id="user-email" placeholder="Enter your email address" type="email"/>
+                    </div>
+                </div>
+                <div>
```

Comment:
```#``` does not serve any purpose here, technically now your label does not match with your inputs id. So you can remove it.

Reply:
made the changes, thanks for the review!